src/twisted/names/dns.py

```diff
@@ -120,8 +123,14 @@ def randomSource():
 A6 = 38
 DNAME = 39
 OPT = 41
+SSHFP = 44
 SPF = 99
 
+# These record types do not exist in zones, but are transferred in
+# messages the same way normal RRs are.
+TKEY = 249
```

Comment:
is TKEY used?

Reply:
Ah, TKEY slipped through from a future PR (I have several new RR implementations to submit but I split them up into smaller pull requests).